bitnami/harbor/conf/clair.yaml

```diff
@@ -0,0 +1,21 @@
+clair:
```

Comment:
Why are you duplicating the values on this file? They are already into the `values.yaml`, Please do no create a new file add the necessary to the `values.yaml`

Reply:
This file comes from moving this out from [clair-cm.yaml](https://github.com/bitnami/charts/pull/1941/files#diff-c1d79c0e047c00a17a428ed4347eb8f8L9-L34)

Reply:
Well, in that case, move those values to the `values.yaml`

Reply:
This is the config file that clair will use [here](https://github.com/bitnami/charts/pull/1941/files#diff-6b752c0b3398da15053a9352f3413995R79), it is not related to values.yaml

Reply:
I'm not clear on this because I think this forces the user to clone the bitnami/charts repository in order to work. Would this file become part of the final tarball?

Reply:
Yes, `helm package` will package the whole directory, except files specified in a `.helmignore`.

This approach is the same as in here, where we import the `cert` folder: https://github.com/bitnami/charts/blob/master/bitnami/harbor/templates/core/core-secret.yaml#L24-L27